app/helpers/availability_helper.rb

```diff
@@ -62,9 +62,11 @@ def document_and_api_merged_results(document, items_list)
   end
 
   def availability_alert(document)
-    document["items_json_display"].map { |item|
-      item["availability"].blank?
-    }.any?
+    unless document["electronic_resource_display"]
```

Comment:
@sensei100,  Thanks. This works, but I'd like to keep `availability_alert` to be a function that will return either `true` or `false`... 

Could we just add `document["items_json_display"].blank? &&`

Reply:
@dkinzer Can you be more specific in where the && would go?  I'm not sure  where to do that since right now we are mapping over a field that potentially doesn't exist.

Reply:
Sure I think
```suggestion
    document["electronic_resource_display"] &&
```
will work.